addons/goggles/script_macros.hpp

```diff
@@ -0,0 +1,36 @@
+#define COMBAT_GOGGLES         ACE_Overlay = QPATHTOF(textures\HUD\CombatGoggles.paa); \
+        ACE_OverlayCracked = QPATHTOF(textures\HUD\CombatGogglesCracked.paa); \
+        ACE_Resistance = 2; \
+        ACE_Protection = 1;
```

Comment:
I know it's just a copy/paste, but this bothers me. Will require all uses of `COMBAT_GOGGLES` to now have a semicolon.

```suggestion
#define COMBAT_GOGGLES \
	ACE_Overlay = QPATHTOF(textures\HUD\CombatGoggles.paa); \
        ACE_OverlayCracked = QPATHTOF(textures\HUD\CombatGogglesCracked.paa); \
        ACE_Resistance = 2; \
        ACE_Protection = 1
```

Reply:
Fine by me, only reason I didn't change it was because I didn't want to change the actual definitions.

Reply:
You didn't make the full change, just the semicolon.

Reply:
I didn't notice it, but the extra line feels unnecessary

Reply:
it's also still indented too far. You should have just committed the suggestion.

Reply:
It didn't let me when you first made it